web/flat-web/deploy/nginx/nginx.conf

```
@@ -15,6 +15,19 @@ http {
     listen 80;
     root /app;
 
+    gzip on;
+    gzip_disable "MSIE [1-6]\.";
+    gzip_min_length 64;
+    gzip_buffers 16 8k;
+    gzip_comp_level 6;
```

```suggestion
    gzip_comp_level 1;
```

Just keep it at 1, 6 will put a lot of stress on the CPU.